### DISCUSSION TITLE
Education account location failed

### DISCUSSION THREAD

**Discussion by Gerald132** [community/community#48701] [GitHub Education]
### Select Topic Area

Bug

### Body

I am unable to get the education access to copilot.

Every time I have tried I get this error:

```

Your application cannot be reviewed until you fix the following:

    Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.

```

However I *have* given access as shown in screenshot.

Have tried phone, laptop, Chrome, Opera, Firefox, in guest or private windows as well as normal ones, and ensure that location is working but always the same error.

I have canceled and retried many times.

Can anybody suggest some way to fix it?

![Screenshot 2023-02-27 194758](https://user-images.githubusercontent.com/117800266/221556450-c932328c-ad25-4c67-a4d4-463a40b0c612.png)

**Comment by Gerald132** [comment]
Unfortunately still can't make this work.  Looking in the Chrome console I can see a "400" error before it tells me that my browser settings are wrong.  But they are not wrong, I have allowed location for github.com.

Can anybody help? I still can't make this work.

![Screenshot 2023-03-08 183241](https://user-images.githubusercontent.com/117800266/223690039-52c0debb-846e-499a-bb2c-09923a7fcdb8.png)

**Comment by cptbtptp01** [comment]
I have occurred the same issue....tried on my MacBook and iPad via chrome, safari, but none of them worked...

**Reply by cptbtptp01** [reply]
ok, after posting my response, I just tried again and somehow it works which doesn't sound logical/reasonable to me... but the steps are as follows:
1- if you have 2 email addresses associated with your account, one personal and one academic, in the 'request a discount - https://education.github.com/discount_requests/application' page, try not selecting your academic email address first
2- Instead, select your personal email address, then you'll find that you are able to type to search your school and add it
3- after selecting your school, it will ask you for a .edu email address
4- select your edu email address
5- repeat step3
6- fill in the third prompt 
7- hit 'continue'

This process works for me, and I hope it works for you as well..
Good Luck!

**Reply by Gerald132** [reply]
Thank you but that didn't work for me

**Comment by Gerald132** [comment]
I still can't make it work after trying again and again.

Now I get this error:

And as beforre, my browser DOES allow location for this site.

'''
Your application cannot be reviewed until you fix the following:
Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
'''

And I HAVE done both of these things, and canceled and tried again, many times.

I tried opening a ticket but they just closed it and said come here.

Can anybody help?

**Comment by noatamir** [comment]
I have the same issue. I tried safari, firefox, and a chromium web browser. I also followed the above procedure with email selection. Nothing works.
